Automattic/jetpack
# Related Posts: Update labels for consistency with core

Fixes #30421.

## Proposed changes:


1. Adds a description to the Related Posts block.
2. Updates the sidebar heading to use sentence case. — "General settings"
3. Updates thumbnail toggle to singular to match other toggles. — "Display thumbnail"
4. Updates layout commands to use sentence case. Matches behaviour in core.

With the above changes, the UI should look like this:

<img width="1011" alt="SCR-20230503-shls" src="https://user-images.githubusercontent.com/40267301/235925421-4f75d13c-f13e-48e1-bd2b-d7c7d634210d.png">

### Other information:


## Jetpack product discussion



pejTkB-yT-p2

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






1. Spin up a test site with this branch.
2. Make sure Related Posts are enabled via Jetpack → Settings → Traffic → Related Posts.
3. Create a new post and add a Related Posts block.
4. Confirm the various labels match the changes listed above (and annotated in the screenshot).
